src/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useEffect, useRef, useState } from 'react';
+import React from 'react';
+
+import { TodoType } from '../types/Todo';
+
+type Props = {
+  todos: TodoType[];
+  addTodo: (newTodoTitle: string) => void;
+};
+
+export const Header: FC<Props> = ({ todos, addTodo }) => {
+  const [newTodoTitle, setNewTodoTitle] = useState<string>('');
+  const newTodoInputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (newTodoInputRef.current) {
+      newTodoInputRef.current.focus();
+    }
+  }, []);
+  const handleNewTodoForm = (e: React.FormEvent) => {
+    e.preventDefault();
+    addTodo(newTodoTitle);
+    setNewTodoTitle('');
+  };
+
+  return (
+    <header className="todoapp__header">
+      <button
+        type="button"
+        className={
+          'todoapp__toggle-all ' +
+          (todos.every(({ completed }) => completed) ? 'active' : '')
+        }
+        data-cy="ToggleAllButton"
+      />
+
+      {/* Add a todos on form submit */}
+      <form onSubmit={handleNewTodoForm}>
+        <input
+          data-cy="NewTodoField"
+          type="text"
+          className="todoapp__new-todos"
```

Comment:
```suggestion
          className="todoapp__new-todo"
```
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/147381375/74593ee3-00b1-4ade-ad09-3f37ade1ea4f)